src/shipit_code_coverage/shipit_code_coverage/codecov.py

```diff
@@ -69,6 +70,9 @@ def generate_info(revision, coveralls_token):
       '-t', 'coveralls',
       '-s', REPO_DIR,
       '-p', '/home/worker/workspace/build/src/',
+      '--service-name', 'TaskCluster',
+      '--service-number', datetime.today().strftime('%Y%m%d'),
+      '--service-job-number', '1',
```

Comment:
Yes, each test suite (mochitest, xpcshell, etc.) will have a different service-job-number (1, 2, etc.). At the moment, the code coverage information from all the test suites is parsed at the same time; in the future information from each test suite will be parsed separately (but in the same task).